benchmarks/benchmarks.json

```diff
@@ -13,19 +13,51 @@
     }
   }},
   {
-  "mlp": {
+  "mlp-fp32-layer": {
     "ref": {
       "type": "C++",
       "benchmark": "bench_mlp",
       "flags": []
     },
     "mlir": {
       "type": "MLIR",
-      "benchmark": "mlp.mlir",
+      "benchmark": "mlp-fp32-1layer-512.mlir",
       "flags": []
     }
   }},
   {
+  "mlp-fp32-3x1024": {
+    "mlir": {
+      "type": "MLIR",
+      "benchmark": "mlp-fp32-3layers-1024.mlir",
+      "flags": ["--disable-lsan"]
+    }
+  }},
+  {
+  "mlp-bf16-3x1024": {
+    "mlir": {
+      "type": "MLIR",
+      "benchmark": "mlp-bf16-3layers-1024.mlir",
+      "flags": ["--disable-lsan"]
+    }
+  }},
+  {
+  "mlp-bf16-10x3584": {
+    "mlir": {
+      "type": "MLIR",
+      "benchmark": "mlp-bf16-10layers-3584.mlir",
+      "flags": ["--disable-lsan"]
+    }
+  }},
+  {
+  "mlp-vnni-3x1024": {
+    "mlir": {
+      "type": "MLIR",
+      "benchmark": "mlp-vnni-3layers-1536.mlir",
+      "flags": ["--disable-lsan"]
+    }
+  }},
+  {
```

Comment:
Small suggestion: mlp tests could be captured together by a high-level `"mlp": {` same as it is done for `matmul`. Imo, it makes the console output a bit nicer with less white-spaces.